Flow.Launcher.Test/Flow.Launcher.Test.csproj

```diff
@@ -49,7 +49,8 @@
 
   <ItemGroup>
     <PackageReference Include="Moq" Version="4.18.4" />
-    <PackageReference Include="nunit" Version="3.14.0" />
+    <PackageReference Include="NUnit" Version="4.1.0" />
```

Comment:
Is it intentional?

Reply:
Oh yes I want to test whether upgrading nunit helps with the ci speed. It is a little bit breaking change but i change all the issue. Since it is only a test runner shouldn't be a problem.